gangplank/ocp/cosa-podman.go

```diff
@@ -248,3 +251,33 @@ func podmanRunner(term termChan, cp CosaPodder, envVars []v1.EnvVar) error {
 	}
 	return nil
 }
+
+// mustHaveImage pulls the image if it is not found
+func mustHaveImage(ctx context.Context, image string) error {
+	uri, _ := os.LookupEnv(podmanContainerHostEnvVar)
+	key, keyDefined := os.LookupEnv(podmanContainerHostSSHEnvVar)
+
+	isRemote := strings.HasPrefix(uri, "ssh://")
+	if isRemote && !keyDefined {
+		return fmt.Errorf("remote podman requires %s to be defined", podmanContainerHostSSHEnvVar)
+	}
```

Comment:
This leaves out support for ssh-agent. Maybe we test the connection with an API call before we error out? Or we could see if the `SSH_AUTH_SOCK` env var is defined and just warn here instead.

Reply:
Indeed. I was looking at the API docs, and didn't realize that Podman supports the SSH Auth socket. Fixed up and repushed.